### PR TITLE
KIALI-2040 Support for snapshot releases through Jenkins Pipeline

### PR DESCRIPTION
**Pull request by israel-hdez**
https://issues.jboss.org/browse/KIALI-2040

### CODE REVIEW

File: Makefile.jenkins [hunk 1]
```diff
@@ -50,20 +75,30 @@ ifdef NPM_TOKEN
 endif
 
 ui-push-version-tag:
+ifeq ($(IS_SNAPSHOT),y)
```
[the remark this answers is not in the document]
Comment (josejulio) [reply Makefile.jenkins]: We usually do a snapshot release on every PR merge, does that mean that for every PR we are going to have an extra commit with the snapshot.X change?

**Comment by israel-hdez** [comment]
No, Travis will still be in charge of those snapshots. This PR is for
Friday’s snapshots.

File: Makefile.jenkins (hunk 1 above)
[the remark this answers is not in the document]
Comment (josejulio) [reply Makefile.jenkins]: Ok, just a side note, the version formats needs to be different than `0.12.0-SNAPSHOT.X` or else we need to ensure a higher number for X, else we can change the travis format.

[the remark this answers is not in the document]
Comment (israel-hdez) [reply Makefile.jenkins]: I though we were comfortable with the current formats.
...at least, this is how it was used in v0.11.0 snapshots and v0.9.0 snapshots (there were no v0.10.0 snapshots).
Look at the latest commits in these tags:
https://github.com/kiali/kiali-ui/tree/v0.11.0-snapshot.1
https://github.com/kiali/kiali-ui/tree/v0.11.0-snapshot.0
https://github.com/kiali/kiali-ui/tree/v0.9.0-snapshot.1
(etc)

Note that for Friday's snapshots, "X" is either "0" or "1". They are indeed released in NPM; see for example: https://www.npmjs.com/package/@kiali/kiali-ui/v/0.11.0-snapshot.1

Not sure if there has been always some confusion about how to do and naming of the Friday's snapshots (which are official pre-releases).

[the remark this answers is not in the document]
Comment (israel-hdez) [reply Makefile.jenkins]: So, if there is something wrong with the naming... I think we should make noise in the dev-list.

[the remark this answers is not in the document]
Comment (josejulio) [reply Makefile.jenkins]: My only concern was for npm. There is no problem if npm is fine with it :)

**Pull request merged**